changelog/14025.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
@raskchanky Catching this in final changelog review, but as a feature this should have the "new feature" formatting, which is on the changelog page on the wiki.

Reply:
Argh, sorry 😞 . I can submit a PR to change it.

Reply:
https://github.com/hashicorp/vault/pull/14326